### PR TITLE
Add `RustShip: a new Rust podcast`

### PR DESCRIPTION
**Pull request by MarcoIeni**

### CODE REVIEW

File: draft/2023-08-23-this-week-in-rust.md [hunk 1]
```diff
@@ -42,6 +42,7 @@ and just ask the editors to select the category.
 ### Research
 
 ### Miscellaneous
+* [RustShip: a new Rust podcast](https://ieni.dev/2023/08/rustship-a-new-rust-podcast/)
```
Comment (bennyvasquez) [inline draft/2023-08-23-this-week-in-rust.md:45]:
```suggestion
* [audio] [RustShip: a new Rust podcast](https://ieni.dev/2023/08/rustship-a-new-rust-podcast/)
```

**Review by bennyvasquez — CHANGES_REQUESTED** [review]
We ask folks to label non-text links with the type of media they are. Suggesting the change here, so you can see what I mean for future submissions!

**Comment by MarcoIeni** [comment]
sure, thanks!

**Comment by bennyvasquez** [comment]
Thank you!

**Pull request merged**